addOns/pscanrules/src/main/resources/org/zaproxy/zap/extension/pscanrules/resources/Messages.properties

```diff
@@ -1,3 +1,4 @@
+pscanruels.csp.nofallback.desc = The Content Security Policy fails to define one of the directives that has no fallback. Missing/excluding them is the same as allowing anything.
```

Comment:
Typo in `pscanrules`, which implies this is not being used (nor covered by the tests).